service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/NotificationHandler.java

```diff
@@ -195,4 +197,21 @@ private void sendCMONotificationForLocalAuthority(final CaseDetails caseDetails,
         sendNotification(CMO_ORDER_ISSUED_CASE_LINK_NOTIFICATION_TEMPLATE, email, localAuthorityNotificationParameters,
             Long.toString(caseDetails.getId()));
     }
+
+    private void sendCMODocumentLinkNotificationForCafcass(final CaseDetails caseDetails,
+                                                           final String localAuthorityCode,
+                                                           final DocmosisDocument document) {
+        Map<String, Object> cafcassParameters;
+        try {
+            cafcassParameters = caseManagementOrderEmailContentProvider.buildCMOIssuedNotificationParametersForCafcass(
+                caseDetails, localAuthorityCode, document);
+
+            String cafcassEmail = cafcassLookupConfiguration.getCafcass(localAuthorityCode).getEmail();
+
+            sendNotification(CMO_ORDER_ISSUED_DOCUMENT_LINK_NOTIFICATION_TEMPLATE, cafcassEmail, cafcassParameters,
```

Comment:
will the sendNotification method not catch the NotificationClientException? why do we need to catch it here also?

Reply:
ah, or is it because the buildCMOIssuedNotificationParametersForCafcass throws a NotificationClientException?

Reply:
yes, the exception is thrown in `CaseManagementOrderEmailContentProvider.buildCommonCMODocumentNotificationParameters`